src/mailer/templates/user-debt-notification.ts

```diff
@@ -68,7 +68,7 @@ Dear ${context.name},
 
 Accoring to our administration, you currently have a balance of ${context.balance.toFormat()}.
 
-Go to the SudoSOS website to deposit money into your account. With this you prevent getting fined into the future.
+Go to the  <a href="https://sudosos.gewis.nl">SudoSOS website</a> to deposit money into your account. With this you prevent getting fined into the future.
```

Comment:
Please use the environment variable `URL` instead of hardcoding the url. You can do this by using the `context.url` variable.